packer/README.md

```diff
@@ -10,6 +10,7 @@
 * jenkins-agent-win2016-x64.json: Windows 2016 Server.
 * jenkins-agent-win2019-x64.json: Windows 2019 Server (Recommended).
 * jenkins-agent-win2019-x64-alpine-wsl.json: Windows 2019 Server with WSL enabled running Alpine 3.
+* jenkins-agent-MacOS-x64.json: MacOS with x86_64_mac os_architecture.
```

Comment:
Can you make everything lower case in the file name just to be consistent.

Reply:
Also add the version of the os such as `macos12`.

Reply:
Pushed in my latest commit @peterzhuamazon